internal/idp/providers/ldap/session.go

```diff
@@ -194,7 +194,7 @@ func trySearchAndUserBind(
 	searchQuery := queriesAndToSearchQuery(
```

Comment:
I'm getting an unused parameter warning on `rootCA` (line 192).

If it's not an error, it can be removed all the way up to `tryBind()`

Reply:
Good catch, I removed the param.